app/app.go

```diff
@@ -2,254 +2,185 @@ package app
 
 import (
 	"fmt"
-	v013 "github.com/bitsongofficial/go-bitsong/app/upgrades/v013"
-	v014 "github.com/bitsongofficial/go-bitsong/app/upgrades/v014"
-	v015 "github.com/bitsongofficial/go-bitsong/app/upgrades/v015"
-	v016 "github.com/bitsongofficial/go-bitsong/app/upgrades/v016"
-	"strings"
-
-	v010 "github.com/bitsongofficial/go-bitsong/app/upgrades/v010"
-	v011 "github.com/bitsongofficial/go-bitsong/app/upgrades/v011"
-	"github.com/bitsongofficial/go-bitsong/x/fantoken"
-	fantokenkeeper "github.com/bitsongofficial/go-bitsong/x/fantoken/keeper"
-	fantokentypes "github.com/bitsongofficial/go-bitsong/x/fantoken/types"
-	"github.com/bitsongofficial/go-bitsong/x/merkledrop"
-	merkledropkeeper "github.com/bitsongofficial/go-bitsong/x/merkledrop/keeper"
-	merkledroptypes "github.com/bitsongofficial/go-bitsong/x/merkledrop/types"
+
+	"github.com/bitsongofficial/go-bitsong/v018/app/keepers"
+	"github.com/bitsongofficial/go-bitsong/v018/app/upgrades"
+	v010 "github.com/bitsongofficial/go-bitsong/v018/app/upgrades/v010"
+	v011 "github.com/bitsongofficial/go-bitsong/v018/app/upgrades/v011"
+	v013 "github.com/bitsongofficial/go-bitsong/v018/app/upgrades/v013"
+	v014 "github.com/bitsongofficial/go-bitsong/v018/app/upgrades/v014"
+	v015 "github.com/bitsongofficial/go-bitsong/v018/app/upgrades/v015"
+	v016 "github.com/bitsongofficial/go-bitsong/v018/app/upgrades/v016"
```

Comment:
I think we should remove all older versions

Reply:
afaik, these are necessary for for the upgrade store keys, so we would run into unexpected issues if these are removed.